### PR TITLE
Update IDETemplateMacros.plist

### PR DESCRIPTION
**Pull request by rcritz**
Implement Scott's idea for automatic copyright year.

### CODE REVIEW

File: MyRWTutorial.xcodeproj/xcshareddata/IDETemplateMacros.plist [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>FILEHEADER</key>
-	<string>/// Copyright (c) 2017 Razeware LLC
+	<string>/// Copyright (c) ___YEAR___ Razeware LLC
```
Comment (sberrevoets) [inline MyRWTutorial.xcodeproj/xcshareddata/IDETemplateMacros.plist:6]: We should also remove the first 2 `//` from this line. Xcode inserts them automatically (thanks Xcode....) so in the current project any new files get `/////`.

**Review by sberrevoets — APPROVED** [review]
Looks good to me!

**Comment by obuseme** [comment]
@sberrevoets can you verify your access now by merging this?

**Comment by sberrevoets** [comment]
Looks like it works :]

**Pull request merged**